src/borg/testsuite/repository.py

```diff
@@ -33,26 +33,24 @@ def remote_repository(tmp_path):
 
 
 def pytest_generate_tests(metafunc):
-    # some tests are run on both local and remote repos
+    # parameterizes tests that run on both local and remote repos
```

Comment:
parametrize (see method name)?

Reply:
I worded it that way because it replaces parameterization in the tests that use it, but maybe "generates" would be more accurate here?